app/views/teams/index.html.erb

```diff
@@ -0,0 +1,10 @@
+<div class='container'>
+  <h3>All Teams:</h3>
+  <ul class='nav nav-pills nav-stacked'>
+    <% @teams.each do |team| %>
```

Comment:
What about extracting this into a partial to keep the view cleaner? That way you can use something like:
````
<%= render(@teams) || t(".no_teams") %>
````
Also might be a good idea for us to start making a habit of using internantionalization for text.